app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,6 @@
+class AdminMailer < ApplicationMailer
+  def admin_notification(user)
+    @user = user
+    mail(to: User.admin.pluck(:email), subject: "New user was registered!")
```

Comment:
```suggestion
    @user = user
    
    mail(
      to:      User.admin.pluck(:email),
      subject: "New user was registered!"
    )
```

старайся дотримуватись такого формату для мейлерів